keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -289,7 +289,8 @@ def __init__(
                 activation=stackwise_activation[stack_index],
                 expansion_index=stack_index,
             )
-            pyramid_level_inputs[stack_index + 2] = x.node.layer.name
+            if stackwise_stride[stack_index] != 1:
```

Comment:
Okay I think this is actually incorrect, and this should be _before_ the inverted_res_block.

Basically, this results in us taking the _first_ output block for a given resolution instead of the last.

As a result, we're seeing `14x14x40` for `p4` on MobileNetv3Small, but we should see `14x14x48`

Reply:
Thanks for the feedback @ianstenbit. I've pushed an update to resolve this and updated the test